httpd/oidc_test.go

```diff
@@ -1005,3 +1005,28 @@ func getPreLoginScriptContent(user dataprovider.User, nonJSONResponse bool) []by
 	}
 	return content
 }
+
+func TestOIDCIsAdmin(t *testing.T) {
+	type test struct {
+		input interface{}
+		want  bool
+	}
+
+	emptySlice := make([]interface{}, 0)
+
+	tests := []test{
+		{input: "admin", want: true},
+		{input: append(emptySlice, "admin"), want: true},
+		{input: append(emptySlice, "user", "admin"), want: true},
```

Comment:
add this

`{input: append(emptySlice, 1), want: false},`

it will crash with the current code and it will work after the suggested change